### PR TITLE
Fix global requirement adjustment box

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Changed global requirement from "temp/O2/ocean" to "global requirements" so that it implies Venus ad well.
- Fixed the link of community-corp wiki page.

image

image

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
This PR addresses #1489.

**Comment by kberg** [comment]
Speaking as someone who doesn't use Venus, I'd much prefer the original. If you wanted to add "Venus" to the global requirements box, that wouldn't bother me as much. Plus, if cards eventually become data driven like Dani is planning, Venus can be optionally added and removed.

File: src/HTML_data.ts [hunk 1]
```diff
@@ -5231,7 +5231,7 @@ export const HTML_DATA: Map<string, string> =
               </div>
               
               <div class="description ">
-                  (Effect: Your temperature, oxygen and ocean requirements are +2 or -2 steps, your choice in each case.)
+                  (Effect: Your temperature, oxygen, ocean, and venus requirements are +2 or -2 steps, your choice in each case.)
```
Comment (alrusdi) [inline src/HTML_data.ts:5234]: Please change this text in all translations too

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/HTML_data.ts (hunk 1 above)
Comment (kberg) [reply src/HTML_data.ts:5234]: "venus requirements" should be "Venus requirements."

Comment (vsrisuknimit) [reply src/HTML_data.ts]: Fixed

**Comment by alrusdi** [comment]
@vsrisuknimit What about Special Design card? Is it included?

**Review by alrusdi — APPROVED** [review]

**Pull request merged**